src/utils/apicalls.jsx

```diff
@@ -63,3 +63,15 @@ export const postCommentByArticleId = (article_id, data) => {
 export const deleteCommentById = (comment_id) => {
   return ncNews.delete(`/comments/${comment_id}`).then((res) => {});
 };
+
+export const getTopics = () => {
+  return ncNews.get("/topics").then((response) => {
+    return response.data.topics;
+  });
+};
+
+export const getTopicArticles = (queries) => {
```

Comment:
It's great that you use an api call to get only the relevant articles but this function looks very similar to the function on line 18. Ideally, to keep code DRY, we should think about if we need a totally new function with very similar functionality or if we can make the one we have more dynamic.